S7.Net/Types/Boolean.cs

```diff
@@ -8,31 +8,12 @@ namespace S7.Net.Types
     public static class Boolean
     {
         /// <summary>
-        /// Returns the value of a bit in a bit, given the address of the bit
+        /// Converts a bit to byte array
         /// </summary>
-        public static bool GetValue(byte value, int bit)
```

Comment:
These are all public methods. We can't just remove these, but we have to mark the old one as obsolete and add the new ones.